<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiafSettingsDialog</class>
 <widget class="QDialog" name="PiafSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Piaf Configuration Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Default measure directory :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Default image directory :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="imagePushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../main/piaf.qrc">
     <normaloff>:/images/16x16/folder-open.png</normaloff>:/images/16x16/folder-open.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Default movie directory :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="Layout1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>550</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer name="Horizontal Spacing2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOk">
      <property name="text">
       <string>&amp;OK</string>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="moviePushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../main/piaf.qrc">
     <normaloff>:/images/16x16/folder-open.png</normaloff>:/images/16x16/folder-open.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="measurePushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../main/piaf.qrc">
     <normaloff>:/images/16x16/folder-open.png</normaloff>:/images/16x16/folder-open.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="defaultMovieLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>490</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="defaultImageLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>490</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveAtExitCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save configuration at exit</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="defaultMeasureLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>490</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="../../../main/piaf.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultImageLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotImageChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultMeasureLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotMeasureChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultMovieLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotMovieChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imagePushButton</sender>
   <signal>clicked()</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotImagePushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>measurePushButton</sender>
   <signal>clicked()</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotMeasurePushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moviePushButton</sender>
   <signal>clicked()</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotMoviePushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAtExitCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>slotSaveAtExit(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PiafSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
